src/calculateRentalCost.js

```diff
@@ -20,7 +20,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const REGULAR_PRICE = days * 40;
```

Comment:
Why uppercase? We use camelcase.
```suggestion
  const regularPrice = days * 40;
```

What is `40`?